cfgov/cal/admin.py

```diff
@@ -28,21 +30,23 @@ def start_date(self, obj):
         return obj.dtstart
     start_date.short_description = 'Start Date'
 
+
 class CFPBPDFFileAdmin(admin.ModelAdmin):
     ordering = ['-id']
 
+
 class CFPBCalendarImportICSFileAdmin(admin.ModelAdmin):
 
     def changelist_view(self, request, extra_context=None):
         context = {}
         context.update(extra_context or {})
-        context.update({ 'icsfile': None,})
-        context.update({ 'calendar_record': None,})
-        context.update({ 'processed_events': None,})
+        context.update({'icsfile': None, })
```

Comment:
Oops, you're right. I made a mistake in how I tested this. My personal preference here would be to eliminate the trailing comma where the closing parenthesis immediately follows. So this is still useful:

```py
data = {
    'foo': 1,
    'bar': 2,
}
```

because then you can add/remove elements without changing extra lines. But in this case:

```py
data = {'foo': 1, 'bar': 2, }
```

the trailing comma isn't really serving any purpose.